pkg/runcfanotify/runcfanotify.go

```diff
@@ -611,37 +766,21 @@ func (n *RuncNotifier) watchRuncIterate() (bool, error) {
 	//   1. from containerd-shim (or similar)
 	//   2. from runc, by this re-execution.
 	// This filter skips the first one and handles the second one.
-	if commFromPid(pid) != "runc" {
-		return false, nil
-	}
-
-	// Parse runc command line
+	comm := commFromPid(pid)
 	cmdlineArr := cmdlineFromPid(pid)
-	createFound := false
-	bundleDir := ""
-	pidFile := ""
-	for i := 0; i < len(cmdlineArr); i++ {
-		if cmdlineArr[i] == "create" {
-			createFound = true
-			continue
-		}
-		if cmdlineArr[i] == "--bundle" && i+1 < len(cmdlineArr) {
-			i++
-			bundleDir = filepath.Join(hostRoot, cmdlineArr[i])
-			continue
-		}
-		if cmdlineArr[i] == "--pid-file" && i+1 < len(cmdlineArr) {
-			i++
-			pidFile = filepath.Join(hostRoot, cmdlineArr[i])
-			continue
-		}
+
+	if len(cmdlineArr) == 0 {
+		return false, nil
 	}
 
-	if createFound && bundleDir != "" && pidFile != "" {
-		err := n.monitorRuncInstance(bundleDir, pidFile)
-		if err != nil {
-			log.Errorf("error monitoring runc instance: %v\n", err)
-		}
+	switch comm {
+	case "conmon":
+		// conmon is a special case because it is not a child of the container
+		n.parseConmonCmdline(cmdlineArr)
```

Comment:
Perhaps I'm asking too much, but it'll be nice to have a short comment indicating how components call each other. Is it something like podman -> conmon -> crun?